osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -186,7 +186,9 @@ private bool addPlayers()
         {
             var playersRequiringPopulation = ladder.Teams
                                                    .SelectMany(t => t.Players)
-                                                   .Where(p => string.IsNullOrEmpty(p.Username) || p.Rank == null).ToList();
+                                                   .Where(p => string.IsNullOrEmpty(p.Username)
+                                                               || p.Country == default
```

Comment:
I think comparing against `Unknown` reads better here.

```suggestion
                                                               || p.Country == Country.Unknown
```